### ISSUE TITLE
a suggestion

### ISSUE THREAD

**Issue by spix18** [McAronDev/Windows-full-speed-HotSpot-for-VR#3]
as title says, the script works great on my AWUS1900 but is there a way to force the hotspot to always use the high speed ? powershell script is kinda not user friendly

**Comment by McAronDev** [comment]
I don't think so. Maybe one day microsoft will fix this. But at this time I don't know any easy method. There is another (but also complicated) way: you can passthrough your usb wifi device to virtual machine with some linux distro installed and set up access point on linux side. So you will just need to start VM to use AP. I tried it using VirtualBox but maybe it is even possible on WSL.
